buildSrc/src/main/groovy/org/elasticsearch/gradle/doc/RestTestsFromSnippetsTask.groovy

```diff
@@ -170,6 +178,7 @@ public class RestTestsFromSnippetsTask extends SnippetsTask {
             current.println('---')
             current.println("setup:")
             body(setup)
+            body(waitForYellow)
```

Comment:
I'd probably just `current.println` the wait_for_yellow stuff. Like, you can even use normal rest-yaml syntax and everything in here.